components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3481,7 +3485,7 @@ def defineAliasesFor(m):
             specs = []
             for constructor in constructors:
                 hook = CONSTRUCT_HOOK_NAME + "_" + constructor.identifier.name
-                name = str_to_const_array(constructor.identifier.name)
```

Comment:
Here we need to change `create_named_constructors`: https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/interface.rs#L324 and then change https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/codegen/CodegenRust.py#L3480